src/platform/plugins/shared/dashboard/public/dashboard_container/component/viewport/dashboard_viewport.tsx

```diff
@@ -101,6 +101,7 @@ export const DashboardViewport = ({
     <div
       className={classNames('dshDashboardViewportWrapper', {
         'dshDashboardViewportWrapper--defaultBg': !useMargins,
+        'dshDashboardViewportWrapper--isFullscreen': fullScreenMode,
```

Comment:
Re-adding this class name after it was removed in https://github.com/elastic/kibana/pull/206991 because we need it for some special hover action styles